package.json

```diff
@@ -65,10 +65,14 @@
     "fsevents": "*"
   },
   "browserslist": [
-    "last 2 versions",
-    "ie 8",
-    "ie 9",
-    "iOS 9"
+    ">0.10%",
```

Comment:
I think I agree only provided that the extra CSS does not have a reasonable fallback.

I've not seen good evidence that by removing these attributes these browsers would receive a broken experience.

For example with flexbox we're talking, 0.01% Safari requests that would get `display: inline-block`

Your example seems hypothetical, since newer versions of Chrome do not even have prefixing as prefixing has been discontinued.

That said, don't think this is worth blocking anything over, so 👍 to whatever you choose to do.

Reply:
All good stuff, I'm really just trying to think out loud here more than anything – trying to make sure that we're not being overly selective.

TBH, I wasn't aware that vendor-prefixing had 'gone away' as much as you (and [MDN](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix)) suggest, so maybe this is a bit academic…